Sources/ContainerCommands/Image/ImageLoad.swift

```diff
@@ -34,14 +34,35 @@ extension Application {
             transform: { str in
                 URL(fileURLWithPath: str, relativeTo: .currentDirectory()).absoluteURL.path(percentEncoded: false)
             })
-        var input: String
+        var input: String?
 
         @OptionGroup
         var global: Flags.Global
 
         public func run() async throws {
-            guard FileManager.default.fileExists(atPath: input) else {
-                print("File does not exist \(input)")
+            let tempFile = FileManager.default.temporaryDirectory.appendingPathComponent("temp-file.tar")
+            if input == nil {
+                guard FileManager.default.createFile(atPath: tempFile.path(), contents: nil) else {
+                    throw ContainerizationError(.internalError, message: "unable to create temporary file")
+                }
+
+                guard let outputHandle = try? FileHandle(forWritingTo: tempFile) else {
+                    throw ContainerizationError(.internalError, message: "unable to open temporary file for writing")
+                }
+
+                let bufferSize = 4096
+                while true {
+                    let chunk = FileHandle.standardInput.readData(ofLength: bufferSize)
+                    if chunk.isEmpty { break }
+                    outputHandle.write(chunk)
+                }
+            }
+            defer {
```

Comment:
Since you're using `try?` you could move this right below L43 to ensure that the file gets removed even if L49 throws.